validation/src/test/scala/hmda/validation/rules/lar/validity/V110Spec.scala

```diff
@@ -0,0 +1,89 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.ts.{ Parent, TransmittalSheet }
+import hmda.model.institution.InstitutionType._
+import hmda.model.institution.{ Agency, Institution, InstitutionType }
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.ts.TsEditCheckSpec
+import org.scalacheck.Gen
+
+class V110Spec extends TsEditCheckSpec {
+
+  private var institution: Institution = _
+
+  override def check: EditCheck[TransmittalSheet] = new V110(institution)
+
+  private val applicableTypes: Set[InstitutionType] = Set(DependentMortgageCompany, Affiliate)
+
+  property("any TS must pass for Institution of type other than MBS or Affiliate") {
+    forAll(tsGen, Gen.oneOf(InstitutionType.values)) { (ts, instType) =>
+      whenever(!applicableTypes.contains(instType)) {
+        whenInstitutionTypeIs(instType)
+        ts.mustPass
+      }
+    }
+  }
+
+  // TODO fix generator so that the previous test will cover this case too. (then remove this one.)
+  property("any TS, even with empty parent info, must pass for Institution of type other than MBS or Affiliate") {
+    forAll(tsGen, Gen.oneOf(InstitutionType.values)) { (ts, instType) =>
+      whenever(!applicableTypes.contains(instType)) {
+        whenInstitutionTypeIs(instType)
+        val noParentTS = ts.copy(parent = Parent("", "", "", "", ""))
+        noParentTS.mustPass
+      }
+    }
+  }
+
+  property("TS must pass if parent info is populated and Institution is dependent MBS or Affiliate") {
+    forAll(tsGen, Gen.oneOf(applicableTypes.toList)) { (ts, instType) =>
```

Comment:
Yes. And it sounds like maybe I can drop "dependent" there, if I understand correctly (from things said elsewhere).